seismostats/plots/statistical.py

```diff
@@ -5,91 +5,59 @@
 from scipy.stats import norm
 
 # Own functions
-from seismostats.analysis.bvalue import BPositiveBValueEstimator, estimate_b
+from seismostats.analysis.bvalue.base import BValueEstimator
+from seismostats.analysis.bvalue.classic import ClassicBValueEstimator
 
 
 def plot_mc_vs_b(
     magnitudes: np.ndarray,
     mcs: np.ndarray,
-    delta_m: float = 0.1,
-    method: str = "classic",
+    delta_m: float,
+    b_method: BValueEstimator = ClassicBValueEstimator,
     confidence_intvl: float = 0.95,
     ax: plt.Axes | None = None,
     color: str = "blue",
+    legend: str = None,
+    **kwargs,
 ) -> plt.Axes:
     """Plots the estimated b-value in dependence of the completeness magnitude.
 
     Args:
         magnitudes: magnitudes of the catalog
         mcs:        completeness magnitudes (list or numpy array)
+        dmc:        if a positive b-value estimator is used, this is the
+                minimum difference that is considered. For the
+                ClassicBValueEstimator, leave this as the default None.
```

Comment:
dmc is not mentioned in the function signature, but passed via the **kwargs, so it makes no sense to have it in the docstring.